### ISSUE TITLE
Start up errors in CentOS 8 with MySQL as the backend

### ISSUE THREAD

**Issue by sandoche2k** [zonemaster/zonemaster-backend#574]
On running the start up 
```
sudo service zm-backend.sh start
Logging to /var/log/zonemaster/zonemaster_backend_testagent.log
zonemaster-testagent                                          [Started]
```

In /var/log/zonemaster/zm-starman-error.log" following message:
```
2019/12/30-10:12:42 Couldn't open pid file "/var/run/zonemaster/zm-starman.pid" [No such file or directory].

  at line 177 in file /usr/share/perl5/vendor_perl/Net/Server.pm
2019/12/30-10:12:42 Received QUIT. Running a graceful shutdown
2019/12/30-10:12:42 Worker processes cleaned up
2019/12/30-10:12:42 Server closing!
```
When I kill /usr/local/bin/zonemaster_backend_testagent and start the backend, I get the following:
```
[centos@ip-172-31-31-205 ~]$ sudo service zm-backend.sh startLogging to /var/log/zonemaster/zonemaster_backend_testagent.log
Use of uninitialized value $pid in numeric eq (==) at /usr/local/share/perl5/Daemon/Control.pm line 267.
Use of uninitialized value $new_pid in numeric eq (==) at /usr/local/share/perl5/Daemon/Control.pm line 270.
```
The program does not exit. But on testing the post sanity check command the backend is working and I get the correct reponse:
```
[centos@ip-172-31-31-205 ~]$ curl -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","method":"version_info","id":"1"}' http://localhost:5000/ && echo
{"id":"1","jsonrpc":"2.0","result":{"zonemaster_engine":"v3.0.3","zonemaster_backend":"5.0.0"}}
```

But the remote server gets hanged. The same scenario has been repeated more than twice.

**Comment by sandoche2k** [comment]
On a new install of CentOS.. everything seems perfect, but testing a domain in the GUI, it is always at 0%. Digging further:
1. Post sanity check is OK
2. In /var/log/zonemaster/zm-starman-error.log
```
Process Backgrounded
2019/12/31-13:35:31 Starman::Server (type Net::Server::PreFork) starting! pid(4991)
Binding to TCP port 5000 on host 127.0.0.1 with IPv4
Setting gid to "990 990"
Setting uid to "993"
```
3. In /var/log/zonemaster/zonemaster_backend_testagent.log
```
ter.conf-example /etc/httpd/conf.d/zonemaster.conf
2019-12-31T13:35:32Z [4992] NOTICE - Loading config: /etc/zonemaster/backend_config.ini
2019-12-31T13:35:32Z [4992] NOTICE - Constructing database adapter: Zonemaster::Backend::DB::MySQL
2019-12-31T13:35:32Z [5001] NOTICE - Daemon spawned
2019-12-31T13:35:32Z [5001] NOTICE - Constructing database adapter: Zonemaster::Backend::DB::MySQL
2019-12-31T13:43:59Z [5001] CRITICAL - DBD::mysql::db selectrow_array failed: Lost connection to MySQL server during query at /usr/local/share/perl5/Zonemaster/Backend/DB.pm line 68.
2019-12-31T13:43:59Z [5001] NOTICE - Daemon terminating
```

**Comment by mattias-p** [comment]
This sounds really similar to #513/#565.

**Comment by matsduf** [comment]
Resolved by #588

**Comment by sandoche2k** [comment]
Closing the issue
